packages/uni-virtualizer/src/lib/VirtualRepeater.ts

```diff
@@ -0,0 +1,631 @@
+import {ItemBox, Margins} from './layouts/Layout'
+
+// The implementer decides the type of children.
+export type Child = any;
```

Comment:
We should try to get rid of all `any`s, which can usually be replaced replace with either `unknown` or generics. Doesn't need to be for this initial PR though necessarily.